Filamento.py

```diff
@@ -0,0 +1,76 @@
+from flask import Flask, render_template, request, redirect, session, flash, url_for
+
+class Filamento:
+    def __init__(self, cor, fornecedor, descricao, qtd):
+        self.cor = cor
+        self.fornecedor = fornecedor
+        self.descricao = descricao      
+        self.qtd = qtd 
+
+filamento1 = Filamento('Vermelho','Volt','Vermelho','1000')
+filamento2 = Filamento('Azul piscina','National','Azul tiffany','500')
+filamento3 = Filamento('Branco','National','Branco gelo','300')
+lista = [filamento1 ,filamento2 ,filamento3]       
+ 
+class Usuario:
+    def __init__ (self, nome, nickname, senha):
+        self.nome = nome
+        self.nickname = nickname
+        self.senha = senha
+       
+usuario1 = Usuario('Luiza', 'Lu', '1234')
+usuario2 = Usuario('Leone', 'Leo', '1235')
+usuario3 = Usuario('Laura', 'Laurita', '1236')
+
+usuarios = { usuario1.nickname :usuario1, 
+                usuario2.nickname :usuario2,
+                usuario3.nickname :usuario3 }
```

Comment:
Imagine it for many users...

How to simplify it with for iteration
```suggestion
users = {}
for usuario in list_users:
    users[usuario.nickname] = usuario
```